src/Common/src/System/Diagnostics/Debug.cs

```diff
@@ -84,19 +125,31 @@ public static void WriteLine(string message)
         [System.Diagnostics.Conditional("DEBUG")]
         public static void Write(string message)
         {
-            s_logger.WriteCore(message ?? string.Empty);
+            if (message == null)
```

Comment:
We previously called WriteCore if message was null, outputting an empty entry.  We don't want to continue that?